Add a response forwarding section to the service documentation (#5517)
containous/traefik: docs/content/routing/services/index.md

                My-Custom-Header: foo
                My-Header: bar
    ```
    
#### Response Forwarding

> Configure response forwarding to define how forwards serves back the response of requests to remote server.

Reviewer: This section is about configuring how Traefik forwards the response from the backend server to the client.

Below are the available options for the Response Forwarding mechanism:

> - `FlushInterval` specifies the flush interval to flush to the client while copying the response body.

Reviewer: ```suggestion
- `FlushInterval` specifies the interval in between flushes to the client while copying the response body.
```

  It is a duration in milliseconds, defaulting to 100.
  A negative value means to flush immediately after each write to the client.

>   The FlushInterval is ignored when ReverseProxy recognizes a response as a streaming response; for such responses, writes are flushed to the client immediately.

Reviewer: ```suggestion
  The FlushInterval is ignored when ReverseProxy recognizes a response as a streaming response;
```

>   

Reviewer: ```suggestion
  for such responses, writes are flushed to the client immediately.
  
```

> ??? example "Use a custom FlushInterval -- Using the [File Provider](../../providers/file.md)"

Reviewer: ```suggestion
??? example "Using a custom FlushInterval -- Using the [File Provider](../../providers/file.md)"
```
